ydb/core/tablet_flat/flat_row_state.h

```diff
@@ -120,6 +122,10 @@ namespace NTable {
             }
         }
 
+        void AddMissingExternalBlobSize(ui32 externalBlobSize) noexcept {
+            MissingExternalBlobsSize_ += externalBlobSize;
```

Comment:
```suggestion
            MissingExternalBlobsSize_ += Max(1, externalBlobSize);
```

Не знаю, вдруг там какие-нибудь пустые блобы